### ISSUE TITLE
ch-image build --force: support SUSE

### ISSUE THREAD

**Issue by reidpr** [hpc/charliecloud#1044]
Specifically openSUSE. There are some quirks:

* Leap (release per year), Tumbleweed (rolling releases, har har har)
* Non-monotonic version numbers (42 < 15).

**Comment by reidpr** [comment]
Waiting on #1042.

**Comment by reidpr** [comment]
Duplicate of #1294.
